GenderPayGap.WebUI/Views/ScopeNew/ConfirmOutOfScope.cshtml

```diff
@@ -0,0 +1,124 @@
+            @(Html.AntiForgeryToken())
+            <h1 class="govuk-heading-xl">Review your information</h1>
+            
+            @(Html.HiddenFor(model => model.WhyOutOfScope))
+            @(Html.HiddenFor(model => model.WhyOutOfScopeDetails))
+            @(Html.HiddenFor(model => model.HaveReadGuidance))
+            @(Html.HiddenFor(model => model.ReportingYear))
+            
+            <table class="govuk-table">
+                <tbody class="govuk-table__body">
+                    <tr class="govuk-table__row">
+                        <th class="govuk-table__header" scope="row">
+                            Reporting Year
+                        </th>
+                        <td class="govuk-table__cell">
+                            @reportingYearFormatted
+                        </td>
+                    </tr>
+                    <tr class="govuk-table__row">
+                        <th class="govuk-table__header" scope="row">
+                            Previous Scope
+                        </th>
+                        <td class="govuk-table__cell">
+                            @Model.Organisation.OrganisationScopes.FirstOrDefault(s => s.SnapshotDate == Model.ReportingYear)?.ScopeStatus
+                        </td>
+                    </tr>
+                    <tr class="govuk-table__row">
+                        <th class="govuk-table__header" scope="row">
+                            New Scope
+                        </th>
+                        <td class="govuk-table__cell">
+                            OutOfScope
+                        </td>
+                    </tr>
+                    <tr class="govuk-table__row">
+                        <th class="govuk-table__header" scope="row">
+                            Reason
+                        </th>
+                        <td class="govuk-table__cell">
+                            @whyOutOfScopeFormatted
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+
+            <p class="govuk-body">
+                You are reminded that it is a legal obligation for relevant employers with 250 or more employees to report.
+                Please ensure that you have considered the appropriate guidance carefully before proceeding.
+                By submitting this notification, you are telling us that your organisation is not subject to the gender
+                pay gap regulations for this reporting year.
+            </p>
+            
+            <p class="govuk-body">
+                The information you provide may be shared with the Equality and Human Rights Commission for the purposes of enforcing compliance with the gender pay gap reporting requirements.
+            </p>
+            
+            @(Html.GovUkButton(
+                new ButtonViewModel
+                {
+                    Text = "Confirm"
+                }
+                ))
+            
+            <p>

```

Comment:
`<p>` is normally done as `<p class="govuk-body">`.
It won't make a lot of difference for this, but if you had any text outside the link it might not show properly.
I tend to always add the `govuk-body` class just to be consistent